src/components/topic/topic.jsx

```diff
@@ -8,181 +8,195 @@ import './topic-style';
-                                key={ currency.name }
-                                className={ `${block}__field` }
-                                currency={ currency }
-                                current={ topic.votes.find(obj => obj.name === currency.name) }
-                                locked={ topic.locked }
-                                user={{
-                                    votes: topic.userVotes && topic.userVotes.find(obj => obj.name === currency.name),
-                                    currency: user && user.currencies.find(obj => obj.name === currency.currency)
-                                }}
-                                onVote={ this._vote.bind(this, currency) } />
-                        ))}
-                        <div className={ `${block}__total` }>
-                            { topic.score }
-                        </div>
-                    </div>
-                </section>
-            </div>
-        );
-    }
-
-    /**
-     * Trigger voting process
-     *
-     * @param {object} settings   - Data describing the type of vote
-     * @param {number} difference - Amount to change influence
-     */
-    _vote(settings = {}, difference) {
-        let { topic } = this.props;
-
-        this.props.onVote(
-            topic.id, 
-            settings.name, 
-            difference, 
-            settings.currency, 
-            settings.score
-        );
-    }
-
-    /**
-     * Enable topic editing
-     * 
-     * @param {object} e - React synthetic event
-     */
-    _edit(e) {
-        this.setState({
-            editing: true
-        });
-    }
-
-    /**
-     * Update the topic's title
-     * 
-     * @param  {object} e - React synthetic event
-     */
-    _changeTitle(e) {
-        this.setState({ 
-            title: e.target.value 
-        });
-    }
-
-    /**
-     * Update the topic's description
-     * 
-     * @param {object} e - React synthetic event
-     */
-    _changeDescription(e) {
-        this.setState({ 
-            description: e.target.value 
-        });
-    }
-
-    /**
-     * Update topic settings
-     * 
-     * @param {object} option - The dropdown object (a label and settings)
-     */
-    _changeSettings({ label, ...settings}) {
-        let { topic } = this.props;
-
-        this.props.onChangeSettings(
-            topic.id,
-            settings
-        );
-    }
-
-    /**
-     * Pass changes up via this.props.save
-     * 
-     * @param {object} e - React synthetic event
-     */
-    _saveChanges(e) {
-        let { topic } = this.props,
-            { title, description } = this.state;
-
-        this.props
-            .onChangeSettings(topic.id, { title, description })
-            .then(success => {
-                if (success) {
-                    this.setState({
-                        editing: false
-                    });
-                }
-            });
-    }
+	constructor(props) {
```

Comment:
Please revert formatting changes, it is unclear what was changed.

Reply:
Done, PTAL. 👍